### PR TITLE
Issue 3202 added 8.2 consumables

### PR DESCRIPTION
**Pull request by leozhekov**
Added the 8.2 consumables and separated the old ones to 'mid tier'. Also added suggestions to use the better ones if 'mid tier' have been used.

### CODE REVIEW

File: src/common/SPELLS/others.js [hunk 1]
```diff
@@ -216,6 +236,51 @@ export default {
     name: 'Bountiful Captain\'s Feast',
     icon: 'inv_cooking_80_majorfeast',
   },
+  FAMINE_EVALUATOR_AND_SNACK_TABLE_FEFAST_INT: {
```
Comment (Sharrq) [inline src/common/SPELLS/others.js]: You have some typos here (FEFAST)

File: src/common/SPELLS/others.js [hunk 2]
```diff
@@ -216,6 +236,51 @@ export default {
...
+    id: 297037,
+    name: 'Bil\'Tong',
+    icon: 'inv_misc_food_48',
+  },
+  FRAGRANT_KAKAVIA: {
+    id: 297040,
+    name: 'Fragrant Kakavia',
+    icon: 'inv_misc_food_159_fish_white',
+  },
+  MECH_DOWEL_BIG_MECH: {
```
Comment (Sharrq) [inline src/common/SPELLS/others.js]: DOWELS not DOWEL

File: src/parser/shared/modules/items/FlaskChecker.js [hunk 3]
```diff
@@ -2,21 +2,43 @@ import Analyzer from 'parser/core/Analyzer';
...
+const MAX_FLASK_IDS = [
+  SPELLS.GREATER_FLASK_OF_THE_CURRENTS.id,
+  SPELLS.GREATER_FLASK_OF_ENDLESS_FATHOMS.id,
+  SPELLS.GREATER_FLASK_OF_THE_UNDERTOW.id,
+  SPELLS.GREATER_FLASK_OF_THE_VAST_HORIZON.id,
+];
+
 class FlaskChecker extends Analyzer {
   startFightWithFlaskUp = false;
+  weakFlaskused = false;
```
Comment (leozhekov) [inline src/parser/shared/modules/items/FlaskChecker.js]: Missed camelCase here. Gonna fix asap.

Comment (leozhekov) [reply src/parser/shared/modules/items/FlaskChecker.js]: Actually, I don't even need this.

**Comment by Sharrq** [comment]
Also it would be helpful if you took some screenshots of the new checklist items and the suggestions. Makes it easier to visually see the changes as you are going through the code.

**Comment by leozhekov** [comment]
> Also it would be helpful if you took some screenshots of the new checklist items and the suggestions. Makes it easier to visually see the changes as you are going through the code.

https://imgur.com/a/uzFAp9J

File: src/common/SPELLS/others.js (hunk 2 above)
Comment (leozhekov) [reply src/common/SPELLS/others.js]: https://ptr.wowhead.com/item=168310/mech-dowels-big-mech

**Comment by Sharrq** [comment]
did you combine some screenshots together or does it really say you didnt pre pot, you didnt use a pot mid fight, but you used a weak potion?

Shouldnt it be either you used a weak one or you didnt at all?

**Comment by leozhekov** [comment]
> did you combine some screenshots together or does it really say you didnt pre pot, you didnt use a pot mid fight, but you used a weak potion?
> 
> Shouldnt it be either you used a weak one or you didnt at all?

I added a new screenshot of a guy using a prepot, but not a high quality one.

**Comment by Sharrq** [comment]
Right but he has a suggestion saying he did not prepot and also one saying he used a weak potion. It cant be both. Either he used a pot but it was a weak one, or he didnt pot.

So it sounds like the "you did not use a potion pre combat" suggestion is incorrectly showing up

**Comment by leozhekov** [comment]
https://imgur.com/a/XavCXdx
The first screenshot is from a SV hunter parse. The guy used a strong prepot but not a second potion during the fight.

**Comment by emallson** [comment]
@Sharrq have you gotten a chance to take another look at this?

**Pull request merged**